source/developer/dev-setup-compiling.rst

```diff
@@ -1,5 +1,22 @@
 .. _dev-compiling:
 
+tl;dr
+-----
+
+.. code-block:: bash
+
+  # Start the API server (required)
+  cd ~/go/src/github.com/mattermost/mattermost-server
+  make run
+
+  # Start the web app (optional)
+  cd $WEBAPP_DIR
+  make help
+
+  # Make docs from either repo
+  make help
+  
+
 Compiling Mattermost
```

Comment:
This section should be modified to mention that you need to do `make run` in both now.

Reply:
What's a sentence in english in your words that would explain what we want a user to know how to do at the end of this section?